ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -52,15 +57,27 @@ function getSubtitleText(exception: VulnerabilityException) {
         case 'APPROVED':
             return `Approved (${numCVEs})`;
         case 'APPROVED_PENDING_UPDATE':
-            return `Approved pending update (${numCVEs})`;
+            return `Approved, pending update`;
         default:
             return ensureExhaustive(exception.status);
     }
 }
 
+export function getCVEsForUpdatedRequest(exception: VulnerabilityException): string[] {
+     if (isDeferralException(exception) && exception.deferralUpdate) {
+         return exception.deferralUpdate.cves;
+     }
+     if (isFalsePositiveException(exception) && exception.falsePositiveUpdate) {
+         return exception.falsePositiveUpdate.cves;
+     }
+     return exception.cves;
+}
+
 function ExceptionRequestDetailsPage() {
+    const [selectedTab, setSelectedTab] = useState<RequestDetailsTab>('REQUESTED_UPDATE');
```

Comment:
Oh yes, I forgot to add a TODO for that. We should do that because users from the `Approved deferrals` or `Approved false positive` table will need it to go to the `Latest approved` tab automatically. Otherwise, it'll look not very clear.